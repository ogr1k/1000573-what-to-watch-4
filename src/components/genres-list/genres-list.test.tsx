import * as React from "react";
import * as renderer from "react-test-renderer";
import GenresList from "./genres-list";

const genres = [`Action`, `Comedy`];
const activeFilter = `Action`;


it(`Render Genres List`, () => {
  const tree = renderer
    .create(<GenresList genres={genres} onFilterClick={() => {}} activeFilter={activeFilter}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
