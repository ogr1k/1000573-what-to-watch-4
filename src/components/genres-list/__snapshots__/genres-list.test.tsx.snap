// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Genres List 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      All Genres
    </a>
  </li>
  <li
    className="catalog__genres-item catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Action
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Comedy
    </a>
  </li>
</ul>
`;
